test/Project.toml

```diff
@@ -0,0 +1,18 @@
+[deps]
+Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+GraphPlot = "a2cc645c-3eea-5389-862e-a155d0052231"
+Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
+IncrementalInference = "904591bb-b899-562f-9e6f-b8df64c7d480"
```

Comment:
IIF as dependency is not needed and can be removed to avoid the whole circular dependency thing.
This test/Project.toml does not have any additional benefit and the normal test section will be supported throughout the julia v1 life cycle.

Reply:
Yeah, with the environment variable it's not needed.